### PR TITLE
changed secretKey and accessKey to be required without quotes

### PR DESCRIPTION
**Pull request by val06**
changed bucketName to be required

### CODE REVIEW

File: helm/seekret/values.yaml [hunk 1]
```diff
@@ -10,11 +10,13 @@ injector:
...
 s3:
-  bucketName: seekret
+  bucketName:
   folderName: pcaps
   ## Default url to communicate with GCS buckets
   s3URL: https://storage.googleapis.com
-  ## Set default accesskey, secretkey
+  ## Use HMAC key auth instead of role based
+  keyAuth: true
+  ## Set default accessKey, secretKey
```
Comment (asdftamir) [inline helm/seekret/values.yaml:19]: update doc

Comment (asdftamir) [reply helm/seekret/values.yaml:19]: and readme

File: helm/seekret/templates/seekret.yaml [hunk 2]
```diff
@@ -175,13 +175,15 @@ data:
...
         value: {{ .Values.s3.s3URL | quote }}
       - name: BUCKET_NAME
-        value: {{ .Values.s3.bucketName }}
+        value: {{required "A target bucket name is required!" .Values.s3.bucketName }}
       - name: FOLDER_NAME
         value: {{ .Values.s3.folderName }}
+      {{ if .Values.s3.keyAuth }}
       - name: ACCESS_KEY
-        value: {{ .Values.s3.accessKey | quote}}
+        value: {{required "An access key ID is required!" .Values.s3.accessKey }}
```
Comment (asdftamir) [inline helm/seekret/templates/seekret.yaml:183]: "if keyAuth is true"

**Review by asdftamir — CHANGES_REQUESTED** [review]

File: helm/seekret/templates/seekret.yaml (hunk 2 above)
Comment (val06) [reply helm/seekret/templates/seekret.yaml:183]: fixed

File: helm/seekret/values.yaml (hunk 1 above)
Comment (val06) [reply helm/seekret/values.yaml]: fixed

**Review by asdftamir — APPROVED** [review]

**Pull request merged**